resources/puppetlabs/lein-ezbake/staging-templates/Dockerfile.mustache

```diff
@@ -0,0 +1,12 @@
+FROM {{{base-image}}}
+
+COPY *.jar /src/
+COPY Dockerfile /Dockerfile
+COPY ext/config/docker /config
+
+VOLUME /ssl
+
+{{{expose-ports}}}
+
+ENTRYPOINT java -Djava.security.egd=/dev/urandom -XX:OnOutOfMemoryError="kill -9 %p" -cp "{{{classpath}}}" clojure.main -m puppetlabs.trapperkeeper.main --config /config/conf.d --bootstrap-config /config/bootstrap.cfg
```

Comment:
One thing that's missing from this is the ability to specify JAVA_ARGS and other variables. If an ezbake config sets specific heap defaults, they're not captured here. Also, how should someone modify them after the fact?

I guess the answer should be: set environment variables when you start the container. Is it still worth including the defaults?

Reply:
It's unclear exactly what you might want to set here. For heap size, we can use some container-specific java flags when we get to java 9/10, so that it will respect the memory limits of the container automatically.